### DISCUSSION TITLE
Provide AppImages for aarch64 Linux

### DISCUSSION THREAD

**Discussion by Azathothas** [ImageMagick/ImageMagick#7690] [Development]
It's easy to use ImageMagick on x86_64 Linux thanks to the prebuilt AppImages.
However, for aarch64, we have to compile from source , if prebuilt AppImages were also for provided aarch64 platform, it would be very appreciated.
Thanks!

**Comment by aik2mlj** [comment]
Second this. Would be great if there are aarch64 Appimages.
